mod-audit-server/src/main/java/org/folio/verticle/AbstractConsumersVerticle.java

```diff
@@ -0,0 +1,73 @@
+package org.folio.verticle;
+
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import org.folio.kafka.AsyncRecordHandler;
+import org.folio.kafka.GlobalLoadSensor;
+import org.folio.kafka.KafkaConfig;
+import org.folio.kafka.KafkaConsumerWrapper;
+import org.folio.kafka.KafkaTopicNameHelper;
+import org.folio.kafka.SubscriptionDefinition;
+import org.folio.okapi.common.GenericCompositeFuture;
+import org.folio.processing.events.utils.PomReaderUtil;
+import org.folio.rest.tools.utils.ModuleName;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class AbstractConsumersVerticle extends AbstractVerticle {
+  private static final GlobalLoadSensor globalLoadSensor = new GlobalLoadSensor();
+
+  @Autowired
+  private KafkaConfig kafkaConfig;
+
+  @Value("${srm.kafka.DataImportConsumer.loadLimit:5}")
```

Comment:
is the name of this config option valid? I dont believe there should be anything relating to data import here.